### ISSUE TITLE
Alt+Tab window switching doesn't work in Puddle

### ISSUE THREAD

**Issue by AdmcWolfy** [ValveSoftware/steam-for-linux#1972]
I know in most fullscreen games in Linux Alt+Tab shortcut doesn't work (this is because of xorg), but in Steam games it works somehow. It works in every game on steam I tested except Puddle. I know I can run Steam in second X session but it's not convenient.

Informacje o procesorze:
    Dostawca: AuthenticAMD
    Szybkość: 3400 MHz
    4 procesory logiczne
    4 procesory fizyczne
    HyperThreading:  Nieobsługiwane
    FCMOV:  Obsługiwane
    SSE2:  Obsługiwane
    SSE3:  Obsługiwane
    SSSE3:  Obsługiwane
    SSE4a:  Obsługiwane
    SSE41:  Nieobsługiwane
    SSE42:  Nieobsługiwane

Informacje o sieci:
    Szybkość sieci:  

Wersja systemu operacyjnego:
    Ubuntu 12.10 (32 bitów)
    Nazwa kernela: Linux
    Wersja kernela: 3.7.7-ext73-f1-22.1-k8-ags-cfs
    Dostawca serwera X: The X.Org Foundation
    Wydanie serwera X: 11300000
    X Window Manager:  KWin
    Steam Runtime Version:  <Runtime enabled>

Karta graficzna:
    Sterownik: NVIDIA Corporation GeForce GTX 550 Ti/PCIe/SSE2/3DNOW!

```
Wersja sterownika: 4.3.0 NVIDIA 313.18
Głębia kolorów pulpitu: 24 bitów na piksel
Częstotliwość odświeżania monitora: 59 Hz
Ident. producenta: 0x10de
ID urządzenia: 0x1244
Liczba monitorów: 1
Liczba logicznych układów graficznych:  1
Rozdzielczość głównego ekranu:  1440 x 900
Rozdzielczość pulpitu: 1440 x 900
Rozmiar głównego ekranu: 16,14" x 10,08" (19,02" diag)
```

41,0cm x 25,6cm (48,3cm diag)
    Szyna: PCI Express 16x
    Ilość pamięci VRAM: 1024 MB
    Obsługiwane tryby MSAA:  2x 4x 8x 16x 

Karta dźwiękowa:
    Dźwięk: Realtek ALC889

Pamięć:
    RAM: 4047 MB

Dodatki:
    Język interfejsu: Polski
    LANG:  pl_PL.UTF-8
    Mikrofon:  Not set
    Dostępne miejsce na twardym dysku: 358073 MB
    Największy wolny blok na twardym dysku: 84007 MB

Zainstalowane oprogramowanie:

Ostatnie raporty niepowodzeń:

**Comment by kisak-valve** [comment]
Hello @AdmcWolfy, are you still experiencing this issue with an up to date system? Can you confirm that this issue was brought to the attention of the game devs?

**Comment by AdmcWolfy** [comment]
Yes, this issue is still present on up-to-date system, the workaround is to simple play this game in windowed mode. I didn't reported this issue to game devs cause I forgot about it.
